app/tests/src/db/models/factories.py

```diff
@@ -80,6 +83,4 @@ class Meta:
     tokens = factory.LazyAttribute(
         lambda o: len(MockSentenceTransformer().tokenizer.tokenize(o.content))
     )
-    mpnet_embedding = factory.LazyAttribute(
-        lambda o: MockSentenceTransformer().encode(o.content)
-    )
+    mpnet_embedding = factory.LazyAttribute(lambda o: MockSentenceTransformer().encode(o.content))
```

Comment:
`make format` made this change, but it shouldn't be needed since `make lint` didn't fail. How can we update `make format` to prevent making this change?

Reply:
Weird, also this wasn't a change in this PR either. I wonder what happened that caused black to put this all in one line? (I'm seeing the same thing on my machine.) If I put it either way on my machine, `make format` does nothing...